src/jaxsim/api/data.py

```diff
@@ -549,6 +446,64 @@ def reset_base_velocity(
             base_linear_velocity=W_v_WB[0:3].squeeze().astype(float),
             base_angular_velocity=W_v_WB[3:6].squeeze().astype(float),
         )
+    
+    def replace(
```

Comment:
Adding `typing.override` decorator would prevent in the future errors related to overriding methods from parent class
```suggestion
    @override
    def replace(
```

Reply:
done!